burst/deploy/burst.yaml

```diff
@@ -18,6 +18,8 @@ spec:
   ports:
   - port: 8080 
     name: http
+    nodePort: 30081
+  type: NodePort # For external access (k3d, kind)
```

Comment:
Why use NodePort instead of routing via selectors?  This seems like we're skipping the benefits of running in k8s.

Reply:
Usually in prod normal k8s service is used and we let k8s figure out the routing.

For dev, we also want to access the service externally. Previously we exported istio LB/Gateway IP and port to an env var `${K8s}`  and put it in ` ~/.bashrc` (see ` patch.sh` ) and thus able to access the service outside the clusters via that exported variable (e.g. `make check`).

With K3d, Kind or minicube it's possible to expose services via yaml config with explicitly selected NodePort. So, we don't have to export and do replacements with sed.

See Kind configuration changes: added exposed port 30080 and 30081
https://github.com/retaildevcrews/istio/pull/43/files#diff-481b8ee3083598968ed95c72ab5c53d3ea7e3b10299a188c8febaaad3617459a